app/services/canonical_pending_transaction_service/settle.rb

```diff
@@ -25,6 +25,11 @@ def run!
           canonical_pending_transaction_id: @canonical_pending_transaction.id,
           canonical_transaction_id: @canonical_transaction.id,
         ).notify_settled.deliver_later
+
+        spending_control = @canonical_transaction.stripe_card.active_spending_control
+        if spending_control.present?
+          SpendingControlService.check_low_balance(spending_control, -@canonical_pending_transaction.amount_cents + @canonical_transaction.amount_cents)
+        end
```

Comment:
I would recommend making this as an `after_create_commit` on CPT as I think this service is ran after a pending transaction settles (so emails will be delayed by a couple of days)

Reply:
I now see how this works - LGTM. I'd recommend passing in a HcbCode and working out the balance off of that.